Lib/sysconfig.py

```diff
@@ -282,6 +282,8 @@ def get_preferred_scheme(key):
 
 
 def get_default_scheme():
+    if sys.prefix != sys.base_prefix:
+        return 'venv'
```

Comment:
We should do this in `get_preferred_scheme`, for `prefix`. Also, we should double-check if this does not false positive on cross builds.

Reply:
Done `get_preferred_scheme`. Not sure how to check cross builds.